index.html

```diff
@@ -87,6 +87,47 @@
         name, card number, etc.) from the end user as an alternative to
         collecting the same information through a [[!HTML]] form.
       </p>
+      <p>
+        The basic card payment method provides information to merchant websites
+        that can be used for multiple transactions over a potentially very long
+        period of time, typically on the order of several years at a time. At
+        the time of the development of this specification, it is commonplace
+        for merchant sites to store this information long-term to reduce the
+        friction of a user entering a credit card number for every future
+        purchase.
+      </p>
+      <p>
+        The decision whether to retain credit card information for future
+        transactions remains a matter of local policy for web sites; however,
+        the introduction of a programmatic way to retrieve credit card
+        information from a web browser affects some key factors that typically
+        motivate storage of such information.
+      </p>
+      <p>
+        Because the web browser will retain credit card information, and make
+        it available – subject to user approval – whenever a merchant needs it,
+        the friction that merchants seek to avoid is reduced. This frees
+        merchants from the liability considerations of storing information on a
```

Comment:
Proposed:

Change:

  "This frees merchants from the liability considerations..."

to:

 "This has the potential to free merchants from some liability considerations...."